client/src/components/reviews/AddReview.jsx

```diff
@@ -1,32 +1,41 @@
 import React from 'react';
+import styled from 'styled-components'
 
-const MODAL_STYLES = {
-  position: 'fixed',
-  top: '50%',
-  left: '50%',
-  transform: 'translate(-50%, -50%)',
-  backgroundColor: '#FFF',
-  padding: '50px',
-  zIndex: 1000,
-};
+const AddReview = ({ showModal, openModal }) => {
+  const CloseButton = styled.button`
+    float: right;
+  `
 
-const OVERLAY_STYLES = {
-  position: 'fixed',
-  top: 0,
-  left: 0,
-  right: 0,
-  bottom: 0,
-  backgroundColor: 'rgba(0, 0, 0, .7)',
-  zIndex: 1000,
-};
+  const MODAL_STYLES = {
```

Comment:
fyi - this is unrelated to Styled Components. Just a style I copied from online for the modal.

Will change this to Styled Components later.